src/journal/journald-server.h

```diff
@@ -186,6 +205,16 @@ struct Server {
         ClientContext *pid1_context; /* the context of PID 1 */
 
         sd_varlink_server *varlink_server;
+
+        /* Caching of credentials data */
+        SocketAddress cred_forward_to_socket;
+        Storage cred_storage;
+
+        JournalCredConfig cred_config;
+        JournalCredConfig cred_config_from_conf;
+
+        JournalCmdlineConfig cmdline_config;
+        JournalCmdlineConfig cmdline_config_from_conf;
```

Comment:
Please also drop Server.max_level_store and introduce one more JournalConfig object for storing final merged config.